docs/modules/ROOT/pages/gsn.adoc

```diff
@@ -0,0 +1,93 @@
+= The Gas Station Network
+
+User onboarding is one of the hottest topics in Ethereum, and meta-transactions the de-facto solution to this important problem. Here you'll learn all about the Gas Station Network (GSN) and how to write contracts that don't require their users to hold Ether to pay for gas.
+
+If you're already up to speed with the workings of the GSN, feel free to skip to <<Receiving a relayed call>>, where we'll go over how to use the OpenZeppelin Contracts to easily write a GSNRecipient contract. Otherwise, strap in!
+
+== Sending gas-less transactions
+
+All Ethereum transactions use gas, and the sender of each transaction must have enough Ether to pay for the gas spent. Even though these gas costs are low for basic transactions (a couple cents), getting this Ether is no easy task: dApp users often need to go through Know Your Customer and Anti Money-Laundering processes (KYC & AML), which not only take time but often involve sending a selfie holding their passport over the Internet (!). Only the most hardcore users will put up with this hassle, and dApp adoption greatly suffers when Ether is required. We can do better.
+
+**Enter meta-transactions**. This is a fancy name for a simple idea: a third-party can send another user's transactions and pay themselves for the gas cost. That's it! There's some tricky technical details, but those can be safely ignored when interacting with the GSN. This way, instead of your users calling into your contract (called the _recipient_) directly, someone else (we'll call them a _relayer_) will send their transaction and pay for the cost.
+
+But why would they do such a thing?
+
+== Incentives
+
+Relayers are not running a charity: they're running a bussiness. The reason why they'll gladly pay for your users' gas costs is because they will in turn charge your contract, the recipient. That way relayers get their money back, plus a bit extra as a _fee_ for their services.
```

Comment:
typo x 2: "bussiness" should be "business"

"Lots of money are" could be "Lots of money is"

"And compared to those, the cost of a couple Ethereum transactions is actually very small" could be
"Compared to those, the cost of a couple of Ethereum transactions is actually very small"